### PR TITLE
add docs for extensions library

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

File: content/en/user-guide/web-application/extensions-library/index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: "Extensions Library"
+weight: 5
+description: Get started with Extensions Library to extend LocalStack by adding new services and features via Extension mechanism.
```
Comment (SimonWallner) [inline content/en/user-guide/web-application/extensions-library/index.md:4]: maybe just a `Extend LocalStack by adding new services and features as extensions`?

File: content/en/user-guide/web-application/extensions-library/index.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+## Introduction
+
+LocalStack Extensions that allow developers to extend and customize LocalStack. A LocalStack Extension is a Python application that runs together with LocalStack within the LocalStack container.
```
Comment (SimonWallner) [inline content/en/user-guide/web-application/extensions-library/index.md:9]: I see the reason behind it, but still would go with the regular word `extension` and thus not capitalize it.

Comment (SimonWallner) [inline content/en/user-guide/web-application/extensions-library/index.md:9]: Localstack extensions ~~that~~ allow...

File: content/en/user-guide/web-application/extensions-library/index.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+LocalStack Extensions are available to the licensed users, and the list of available extensions is available in the [Extensions Library](https://app.localstack.cloud/extensions/library).
```
Comment (SimonWallner) [inline content/en/user-guide/web-application/extensions-library/index.md:11]: to ~~the~~ licensed users

Comment (SimonWallner) [inline content/en/user-guide/web-application/extensions-library/index.md:11]: list of available extensions _can be found_ in the ...

File: content/en/user-guide/web-application/extensions-library/index.md (hunk 2 above)
Comment (SimonWallner) [reply content/en/user-guide/web-application/extensions-library/index.md:9]: I'm ok with `Extensions Library` though. in the app we had similar thoughts when it comes to `auth tokens` or `Auth Tokens` for that matter but in the end settled with the former.

File: content/en/user-guide/web-application/extensions-library/index.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+<img src="extensions-library-ui.png" alt="LocalStack Extensions Library" title="LocalStack Extensions Library" width="900" />
+
+## Installing an Extension
+
+To install an Extension using the LocalStack Extensions Library, you can navigate to the [**app.localstack.cloud/extensions/library**](https://app.localstack.cloud/extensions/library) and click on the **Go to Instance** button to open the list of available instances. If you are running your LocalStack instance locally, you can click on the **Default** option.
+
+You will be redirected to the LocalStack Instance page, where you can directly click the **Install** button to install the Extension. The installation process will take a few seconds, and will restart your LocalStack instance. Click **Continue** to proceed.
```
Comment (SimonWallner) [inline content/en/user-guide/web-application/extensions-library/index.md:19]: same concern as above, would lowercase `instance`

Comment (SimonWallner) [inline content/en/user-guide/web-application/extensions-library/index.md:19]: more emphasis should be put on 'will restart your LS instance' might be worth to put that into an alert box or similar. just so users know.

File: content/en/user-guide/web-application/extensions-library/index.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+You can further manage the installed Extensions by navigating to the **Extensions** tab in the LocalStack Instance page. You can remove an Extension by clicking the **Remove** button.
+
+<img src="extensions-library-management.png" alt="Installed LocalStack Extensions Library" title="Installed LocalStack Extensions Library" width="900" />
+
+### Supported Extensions
+
+The following Extensions are currently available in the Extensions Library:
+
+-   [AWS Replicator](https://docs.localstack.cloud/user-guide/tools/localstack-extensions/aws-replicator-extension/)
```
Comment (SimonWallner) [inline content/en/user-guide/web-application/extensions-library/index.md:31]: a 1 sentence description of each could be helpful here.

File: content/en/user-guide/web-application/extensions-library/index.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+-   [Stripe](https://docs.localstack.cloud/user-guide/tools/localstack-extensions/stripe-extension/)
+-   [Miniflare](https://docs.localstack.cloud/user-guide/tools/localstack-extensions/miniflare-extension/)
+-   [MailHog](https://docs.localstack.cloud/user-guide/tools/localstack-extensions/mailhog/)
+-   [httpbin](https://docs.localstack.cloud/user-guide/tools/localstack-extensions/httpbin-extension/)
+-   [Diagnostic Viewer](https://docs.localstack.cloud/user-guide/tools/localstack-extensions/diagnosis-viewer-extension/)
```
Comment (SimonWallner) [inline content/en/user-guide/web-application/extensions-library/index.md:36]: maybe round it out with a 
`## Create your own extension`
section that gives basic instructions or provides further links.

**Review by SimonWallner — APPROVED** [review]
very nice! left a few remarks on language and capitalization, but since I'm even less a native speaker your guess is probably better than mine, so take my comments with a good pinch of salt.

File: content/en/user-guide/web-application/extensions-library/index.md (hunk 6 above)
Comment (HarshCasper) [reply content/en/user-guide/web-application/extensions-library/index.md]: Will do once this PR is merged https://github.com/localstack/docs/pull/750

**Pull request merged**